src/options/options.html

```diff
@@ -81,8 +81,11 @@ <h6 class="margin-top-medium vocab heading">All Providers</h6>
           </div>
         </div>
         <div class="content" data-content-no="2">
-          <!-- Persist search even when extension is closed?
-          yes (checkbox) -->
+          <h6 class="vocab heading">Options</h6>
+          <div class="use-case filter-checkboxes">
```

Comment:
You have copy pasted these from above. These styles may be needed, but you must change the names of classes, or make a general one that will be used by all checkboxes wrapper.